libs/coin-modules/coin-casper/package.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "@ledgerhq/coin-casper",
+  "version": "1.3.2",
+  "description": "Ledger Casper integration",
+  "keywords": [
+    "Ledger",
+    "LedgerWallet",
+    "Casper",
+    "CSPR"
```

Comment:
Does it make any difference to use `cspr` or `CSPR` ? Cause in every other coin-modules it is in `lowercase`